pydantic/main.py

```diff
@@ -232,14 +232,20 @@ def __pydantic_init_subclass__(cls, **kwargs: Any) -> None:
 
     @classmethod
     def model_validate(
-        cls: type[Model], obj: Any, *, strict: bool | None = None, context: dict[str, Any] | None = None
+        cls: type[Model],
+        obj: Any,
+        *,
+        strict: bool | None = None,
+        from_attributes: bool | None = None,
+        context: dict[str, Any] | None = None,
     ) -> Model:
         """Validate a pydantic model instance.
 
         Args:
             cls: The model class to use.
             obj: The object to validate.
             strict: Whether to raise an exception on invalid fields. Defaults to None.
+            from_attributes (bool | None, optional): Whether to extract data from object attributes. Defaults to None.
```

Comment:
To be in same format with other args
```suggestion
            from_attributes: Whether to extract data from object attributes. Defaults to None.
```